packages/client/src/graphql/fragments.graphql

```diff
@@ -420,3 +420,36 @@ fragment CommonPaginatedResultInfo on PaginatedResultInfo {
   next
   totalCount
 }
+
+fragment RevenueAggregate on RevenueAggregate {
+  __typename
+  total {
+    ...Erc20Amount
+  }
+}
+
+fragment PublicationRevenue on PublicationRevenue {
+  __typename
+  publication {
+    ... on Post {
+      ...Post
+    }
+
+    ... on Mirror {
+      ...Mirror
+    }
+
+    ... on Comment {
+      ...Comment
+    }
+  }
+
+  ...Revenue
+}
+
+fragment Revenue on PublicationRevenue {
```

Comment:
are these fragments used anywhere outside of the `revenue` module? if not, then let's keep them together with queries in the modules .graphql file